### PR TITLE
Add content to tutorials docs page.

### PR DESCRIPTION
**Pull request by alexfogg**
Follow up to CE RKO breakout - adding docs content to a new user-focused tutorials / getting started page as a landing spot for async learning & enablement.

## Test plan

Docs change only, tested locally.

### CODE REVIEW

File: doc/tutorials/index.md [hunk 1]
```diff
@@ -1,93 +1,73 @@
...
+| [Using the Sourcegraph CLI](https://docs.sourcegraph.com/cli/quickstart) | How-To-Guide | `src` is a command line interface to Sourcegraph that allows you to search code from your terminal, create and apply batch changes, and manage and administrate repositories, users, and more. |
+| [Saving Searches](https://docs.sourcegraph.com/code_search/how-to/saved_searches) | How-To-Guide | Saved searches let you save and describe search queries so you can easily find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories. |
+| [Search Contexts](https://docs.sourcegraph.com/code_search/how-to/search_contexts) | How-To-Guide | Search contexts help you search the code you care about on Sourcegraph. A search context represents a set of repositories at specific revisions on a Sourcegraph instance that will be targeted by search queries by default.
+ |
```
Comment (amenne) [inline doc/tutorials/index.md:34]: I think because this is on its own line it's just showing up as text

**Review by amenne — APPROVED** [review]
One tiny thing but this is fantastic and a great starting point to build on! Merge it!

**Comment by alexfogg** [comment]
Ooh @amenne one last thing - we need to put a link in the sidebar somewhere. What do you think? Idea in screenshot below.

Screen Shot 2023-02-16 at 9 47 15 AM

**Comment by alexfogg** [comment]
@amenne pushed up a few things

- Added sidebar link "Tutorials"
- Changed "Getting Started" to "Home"
- Linked to 101 and Tours from within the tutorials page
- Misc cleanup (relative link paths, typos)

Just to get this shipped faster I decided to not rename tours right now since it's linked from a couple spots - just wanna get this out the door.

**Pull request merged**